package.json

```diff
@@ -18,11 +18,9 @@
   "homepage": "https://github.com/CirclesUBI/circles-contracts",
   "devDependencies": {
     "bignumber.js": "^8.1.1",
+    "chai": "^4.1.2",
     "chai-bignumber": "^2.0.2",
     "chai-bn": "^0.1.1",
-    "eslint": "^6.2.2",
```

Comment:
Maybe you have to rebase with `master`? Looks like these dependencies should not be removed?